app/controllers/comments_controller.rb

```diff
@@ -187,7 +187,7 @@ def thread_followers
   end
 
   def thread_rename
-    if @comment_thread.read_only? && !current_user.is_moderator
+    if @comment_thread.read_only? && !current_user.at_least_moderator?
```

Comment:
Untested suggestion:

```suggestion
    if @comment_thread.read_only? && current_user.standard?
```

Reply:
I believe `current_user` is safe to switch, but let's avoid relying on its implementation too much just in case.